### PR TITLE
Portfolio mobile menu

### PR DESCRIPTION
**Pull request by solog0039**
In this project:

1. Github work flow created
2. Linters installed
3. Mobile menu created

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+  <title>My portifolio</title>
+  <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+  <header class="toolbar">
+    <a href="index.html" class="name button"> SOLOMON</a>
+    <nav>
+      <img  class="huberger" onclick="openNav()" src="/assets/Union.png" alt="Dropdown menu">
```
Comment (tugsem) [inline index.html]: - [ ] Your hamburger button isn't visible. Kindly try to update your relative source path to fix it.
image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,1408 @@
...
+
+.humberger {
+  display: flex;
+  flex: 1;
+  padding-right: 12px;
+}
+
+.humberger:hover {
+  color: white;
+}
```
Comment (tugsem) [inline style.css:240]: - [ ] Please show attention to your class names and your CSS selectors. There aren't any elements with this name in your HTML.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const myNavv = document.getElementById('myNav');
+function openNav() {
+  myNavv.style.width = '100%';
+}
+function closeNav() {
+  document.getElementById('myNav').style.width = '0%';
+}
+const openNavv = document.getElementById('.myNav');
```
Comment (tugsem) [inline index.js:8]: - [ ] Kindly remove scrolling from your mobile menu when it's open. That would look better.
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+    </nav>
+    <div id="myNav" class="m-menu">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+      <div class="m-menu-content">
+        <a href="#section-01" onclick="closeNav()">Portfolio</a>
```
Comment (tugsem) [inline index.html:22]: - [ ] Your portfolio link isn't opening the portfolio section. It shows the headline. Kindly update your href value to fix this.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,1408 @@
...
+  background: #3c3a39;
+  padding: 5%;
+  min-height: 5vh;
+}
+
+/* menu strt */
+.m-menu {
+  height: 100%;
+  width: 0;
+  position: fixed;
```
Comment (tugsem) [inline style.css:190]: - [ ] Your menu doesn't fix to the top right now. Kindly fix this issue to help readers.

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @solog0039  and @smunoz1988 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️linters are passing
✔️correct use of github flow
✔️professional documentation
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] Please attempt to provide equal commits next time in a pair-programming activity.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @solog0039 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- Professional PR :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**